tools/cldr-code/src/main/java/org/unicode/cldr/util/MemVoterReportStatus.java

```diff
@@ -1,19 +1,24 @@
 package org.unicode.cldr.util;
 
+import java.util.Date;
 import java.util.HashMap;
 import java.util.Map;
 
 /**
  * Trivial, in-memory implementation of a VoterReportStatus<> and ReportStatusUpdater<>
  */
-class MemVoterReportStatus<T extends Comparable<T>> extends VoterReportStatus<T> implements ReportStatusUpdater<T> {
+public class MemVoterReportStatus<T extends Comparable<T>> extends VoterReportStatus<T> implements ReportStatusUpdater<T> {
     Map<Pair<T, CLDRLocale>, ReportStatus> data = new HashMap<>();
 
     @Override
     public ReportStatus getReportStatus(T user, CLDRLocale locale) {
         return data.computeIfAbsent(Pair.of(user, locale), k -> new ReportStatus());
     }
 
+    public void markReportComplete(T user, CLDRLocale locale, ReportId r, boolean marked, boolean acceptable, Date date) {
```

Comment:
six parameters is a lot, might improve with new class for parameter

Reply:
could be.  it has only two call sites, though, so I don't expect the API to have wide use.